blazor/datagrid/editing.md

```diff
@@ -1225,6 +1225,61 @@ This is demonstrated in the following sample code:
 
 > You can find the fully working sample [here](https://github.com/SyncfusionExamples/blazor-datagrid-different-editmode-for-add-edit-action).
 
+## Update boolean column value in single click
+
+You can make the boolean column editable with a single click in the normal mode of editing by using the column template feature of the Grid.
+
+In the following sample, boolean values as checkboxes in the Verified column are rendered by setting the [DisplayAsCheckBox](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_DisplayAsCheckBox) property to true. To make them editable with a single click, the `SfCheckbox` component is rendered as a template in the Verified column.
```

Comment:
In the following sample, boolean value of Verified column is displayed as checkbox by setting the [DisplayAsCheckBox]

Reply:
Yes, the DisplayAsCheckBox property is not needed. because we render the Sfcheckbox in the column template. So I removed that property and changed the content accordingly.